File: core/data/tests/serialize_anonymous_field_as/output.kt
@file:NoLiveLiterals

package com.agilebits.onepassword

import androidx.compose.runtime.NoLiveLiterals
import kotlinx.serialization.*

@Serializable
sealed class SomeEnum {
	/// The associated String contains some opaque context
	@Serializable
	@SerialName("Context")
	data class Context(val content: String): SomeEnum()
	@Serializable
	@SerialName("Other")
	data class Other(val content: Int): SomeEnum()
}

File: core/data/tests/boxed_value/output.scala
package com.agilebits

package onepassword {

// This is a comment.
sealed trait Colors {
	def serialName: String
}
object Colors {
	case object Red extends Colors {
		val serialName: String = "Red"
	}
	case object Blue extends Colors {
		val serialName: String = "Blue"
	}
	case class Green(content: String) extends Colors {
		val serialName: String = "Green"
	}
}

}
